packages/cli/commons-dev/package.json

```diff
@@ -16,7 +16,10 @@
     "./.prettierrc": "./.prettierrc.js",
     "./.prettierrc.js": "./.prettierrc.js",
     "./tsconfig.json": "./tsconfig.json",
-    "./*": "./lib/*"
+    "./*": {
+      "types": "./lib/*.d.ts",
+      "default": "./lib/*.js"
+    }
```

Comment:
https://nodejs.org/api/packages.html#community-conditions-definitions
https://nodejs.org/api/packages.html#conditional-exports

TL:DR:
- TS resolve `@coveo/cli-commons-dev/what/ever` as `@coveo/cli-commons-dev/lib/what/ever.d.ts`
- JS resolve `@coveo/cli-commons-dev/what/ever` as `@coveo/cli-commons-dev/lib/what/ever.js`


Similar for `@coveo/cli-commons`